tabbycat/options/templates/preferences_index.html

```diff
@@ -49,6 +49,18 @@
       {% trans "How adjudicators are ranked and who can submit feedback" as description %}
       {% include "components/item-action.html" with type="primary" emoji="🙅" text=title extra=description %}
 
+      {% if show_leagues %}
+        {% tournamenturl 'options-tournament-section' section='league_options' as url %}
+        {% trans "League Options" as title %}
+        {% trans "Special options for running large multi-venue multi-level debating competitions. Warning: these features are not fully tested or supported." as description %}
+        {% include "components/item-action.html" with type="primary" emoji="🏫" text=title extra=description %}
+      {% endif %}
```

Comment:
Why is this block being shown as added? It is in `develop`.

Reply:
The 'league' class of preferences were a commissioned set of features for running a large-scale high school league tournament; however they deviate significantly from how 'standard' tournaments work and have some incompatibilities with the standard preferences. They are also at this stage not fully tested with some changes in the `2.X` branch. For this reason we have basically hidden them behind a flag so that normal users don't come across them.

Reply:
It's being shown as added because I moved it from one side to the other - it made more sense in terms of UI balance if we ever enable it.

Reply:
So it's duplicated on line 95.

Reply:
Oops - that's an accidental artefact of the manual merge I had to do. I'll fix it in a commit in the next few hours.